.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -73,10 +73,11 @@ body:
     id: env-details
     attributes:
       label: Environment details
-      description: Please provide any relevant environment details
+      description: Please specify installation info and paste the results from print_env.sh here
       placeholder: |
-       Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
-       If method of install is Docker, provide `docker pull` & `docker run` commands used
+       + Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
+       + If method of install is Docker, provide `docker pull` & `docker run` commands used
+       + print_env.sh output
```

```suggestion
       Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
       If method of install is Docker, provide `docker pull` & `docker run` commands used
       print_env.sh output
```

Are these `+` characters intended?

---

Thank you!

Yeah they were intended just as a bullet between the lines because you can't actually do markdown in the placeholder text to make a real bulleted list.

Not married to them though!

---

No problem. just checking! Didn't know if it was a copy/paste error.

---

I think the bullets are useful. However `print_env.sh output` is not very instructive.   I think it should at least be a link to the .sh file in the repo.

---

```suggestion
       + Environment location: [Bare-metal, Docker, Cloud(specify cloud provider)]
       + If method of install is Docker, provide `docker pull` & `docker run` commands used
       + Run [print_env.sh](https://github.com/rapidsai/cuspatial/blob/main/print_env.sh) and paste in the output.
```

---

Right now, in the placeholder text the link wouldn't actually work since placeholder text doesn't support markdown. 

I've added in a link in the description and referenced it in the placeholder, I think that should be good. Thoughts @harrism?